uxarray/grid/grid.py

```diff
@@ -1582,6 +1588,13 @@ def is_subset(self):
         """Returns `True` if the Grid is a subset, 'False' otherwise."""
         return self._is_subset
 
+    @property
+    def max_face_radius(self):
+        """Maximum face radius of the grid in radians"""
+        if "max_face_radius" not in self._ds:
+            self._ds["max_face_radius"] = _populate_max_face_radius(self)
+        return self._ds["max_face_radius"]
```

Comment:
Can we store this as degrees? It looks like we convert it radians anyways in the function call.